drivers/crc/Kconfig.stm32

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2024 Brill Power Ltd.
+# SPDX-License-Identifier: Apache-2.0
+
+config CRC_STM32
+	bool "STM32 CRC driver"
+	default n
+	depends on DT_HAS_ST_STM32_CRC_ENABLED
+	select USE_STM32_LL_CRC
```

Comment:
`USE_STM32_LL_F00` are required only when making use of c LL files, in your case unless you use `LL_CRC_DeInit` function, inlcuding `stm32_ll_crc.h` is sufficient to use the API.